api/core/workflow/nodes/start/start_node.py

```diff
@@ -24,3 +31,54 @@ def _run(self) -> NodeRunResult:
         outputs = dict(node_inputs)
 
         return NodeRunResult(status=WorkflowNodeExecutionStatus.SUCCEEDED, inputs=node_inputs, outputs=outputs)
+
+    def _validate_and_normalize_json_object_inputs(self, node_inputs: dict[str, Any]) -> None:
+        for variable in self.node_data.variables:
+            if variable.type != VariableEntityType.JSON_OBJECT:
+                continue
+
+            key = variable.variable
+            value = node_inputs.get(key)
+
+            # 1. 必填校验
+            if value is None:
+                if variable.required:
+                    raise ValueError(f"{key} is required in input form")
+                # 非必填且没传，直接跳过
+                continue
+
+            # 2. 支持前端传字符串或已经是对象
+            if isinstance(value, str):
+                try:
+                    parsed = json.loads(value)
+                except json.JSONDecodeError:
+                    raise ValueError(f"{key} must be a valid JSON object")
+                value = parsed
+
+            if not isinstance(value, dict):
+                # 这里只允许 object，数组之类可以后续再扩展
+                raise ValueError(f"{key} must be a JSON object")
+
+            # 3. JSON Schema 校验（可选）
+            schema = getattr(variable, "json_schema", None)
+
+            if schema:
+                # schema 既可能是字符串也可能已经是 dict
+                if isinstance(schema, str):
```

Comment:
I suggest requiring JSON schema to be a JSON object. This simplifies code and provider clear semantics. 

The `VariableEntity` is a Pydantic `BaseModel`. The type of `json_schema` field could be automatically verified by relevant methods.